api/pkg/s3/objectpartcopy.go

```diff
@@ -0,0 +1,170 @@
+// Copyright 2019 The OpenSDS Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package s3
+
+import (
+	"net/url"
+	"strconv"
+	"strings"
+
+	"github.com/emicklei/go-restful"
+	"github.com/journeymidnight/yig/helper"
```

Comment:
as this helper and such functions become core part within multicloud as native integration, should we move the helper code to multicloud github as package and then import?

Reply:
there is already helper in https://github.com/opensds/multi-cloud/s3/pkg/helper

Reply:
@skdwriting You are right, this should be s3/pk/helper.

Reply:
@sunfch Could you please take a look of this?

Reply:
@skdwriting @sfzeng , sorry for my mistake.  I fix it right now, thanks